video/out/vo_gpu_next.c

```diff
@@ -1115,6 +1115,10 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         }
     }
 
+#if PL_API_VER >= 330
+    pl_frames_infer_mix(p->rr, &mix, &target, NULL);
```

Comment:
I wonder if we could also figure out a way to readback the inferred image parameters, which would finally solve those complains about DoVi metadata not being exposed via properties

Reply:
> DoVi metadata not being exposed via properties

What DV metadata you mean exactly? I can take a look, hmm...

Reply:
Actually nvm, you are right I can remove custom code for getting ref frame.

Reply:
done